packages/editor-ui/src/stores/workflows.store.ts

```diff
@@ -1312,6 +1313,12 @@ export const useWorkflowsStore = defineStore(STORES.WORKFLOWS, {
 			forceSave = false,
 		): Promise<IWorkflowDb> {
 			const rootStore = useRootStore();
+
+			if (data.settings === null) {
+				EventReporter.info('Detected workflow payload with settings as null', { extra: { data } });
```

Comment:
I'm afraid capturing `data` here can lead to sensitive information being collected wdyt? Especially with `nodes` where parameters might be exposed, including pinned data. The stack trace IMO is the most valuable information we can extract here, so we can pinpoint where this came from.